# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../barnyard2/inline.patch
# Copyright (C) 2010 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

Don't see a header on my raw packets.

  - Rene Rebe <rene@exactcode.de>

--- barnyard2-1.8/src/decode.h.vanilla	2010-06-13 21:00:46.000000000 +0200
+++ barnyard2-1.8/src/decode.h	2010-06-13 21:01:03.000000000 +0200
@@ -191,7 +191,7 @@
 #endif
 
 /* NULL aka LoopBack interfaces */
-#define NULL_HDRLEN             4
+#define NULL_HDRLEN             0
 
 /* enc interface */
 struct enc_header {
